static/src/javascripts/projects/common/modules/experiments/tests/contributions-embed.js

```diff
@@ -23,102 +25,106 @@ define([
              robust,
              giraffeMessage,
              arrowRight,
-             config
+             config,
+             embed,
+             spaceFiller
 ) {
 
 
     return function () {
 
-        this.id = 'ContributionsEmbed20160823';
-        this.start = '2016-08-23';
-        this.expiry = '2016-09-06';
+        this.id = 'ContributionsEmbed20160902';
+        this.start = '2016-09-02';
+        this.expiry = '2016-09-07';
         this.author = 'Jonathan Rankin';
-        this.description = 'Test contributions embed with amount picker.';
+        this.description = '    "Test whether contributions embed performs better inline and in-article than at the bottom of the article.",';
         this.showForSensitive = false;
         this.audience = 0.10;
-        this.audienceOffset = 0.13;
+        this.audienceOffset = 0.23;
         this.successMeasure = 'Impressions to number of contributions';
         this.audienceCriteria = 'All users';
         this.dataLinkNames = '';
-        this.idealOutcome = 'The interactive embed performs 3x better than the control';
+        this.idealOutcome = 'The embed performs 20% better inline and in-article than it does at the bottom of the article';
         this.canRun = function () {
             var pageObj = config.page;
             return !(pageObj.isSensitive || pageObj.isLiveBlog || pageObj.isFront || pageObj.isAdvertisementFeature) && pageObj.edition === 'UK';
         };
 
+        function getSpacefinderRules() {
+            return {
+                bodySelector: '.js-article__body',
+                slotSelector: ' > p',
+                minAbove: 250,
+                minBelow: 100,
+                clearContentMeta: 50,
+                selectors: {
+                    ' .element-rich-link': {minAbove: 100, minBelow: 100},
+                    ' .player': {minAbove: 200, minBelow: 200},
+                    ' > h2': {minAbove: 200, minBelow: 0},
+                    ' > *:not(p):not(h2):not(blockquote)': {minAbove: 35, minBelow: 200},
+                    ' .ad-slot': {minAbove: 150, minBelow: 200}
+                }
+            };
+        }
 
-        var writer = function (component) {
+        var bottomWriter = function (component) {
 
             return fastdom.write(function () {
                 var a = $('.submeta');
                 component.insertBefore(a);
```

Comment:
Injecting things right before the submeta may cause issues with Outbrain, it may be worth checking that all is okay or adding to the Outbrain config: https://github.com/guardian/frontend/pull/13845